kafka-connect-s3/src/main/java/io/confluent/connect/s3/S3SinkTask.java

```diff
@@ -120,7 +127,7 @@ public void open(Collection<TopicPartition> partitions) {
     assignment.addAll(partitions);
     for (TopicPartition tp : assignment) {
       TopicPartitionWriter writer =
-          new TopicPartitionWriter(tp, storage, writerProvider, partitioner, connectorConfig, context);
+          new TopicPartitionWriter(tp, writerProvider, partitioner, connectorConfig, context, time);
```

Comment:
I was thinking that a PR to clean some unused code would make sense to be issued separately. There are some unused arguments here and there. Since this targets 3.2.x I wanted to keep noise to the very minimum